packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -2,6 +2,8 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+
+// eslint-disable-next-line import/no-nodejs-modules
 import * as querystring from "querystring";
```

Comment:
I think we can do:

```suggestion

// Just using the type from a node.js module does not introduce a runtime dependency:
// eslint-disable-next-line import/no-nodejs-modules
import type { ParsedUrlQueryInput } from "querystring";
```

Then update the use below accordingly.